client/src/components/Form/PasswordInput/index.jsx

```diff
@@ -24,7 +23,7 @@ const PasswordInput = ({ verify, passwordToMatch, ...props }) => {
       icon="lock"
       placeholder="Password"
       type="password"
-      pattern={pattern}
+      // pattern={pattern}
```

Comment:
I have left a todo comment above to uncomment this before production. Since this is in testing, we have to create and remember unnecessary long passwords.